sdcm/sct_config.py

```diff
@@ -1420,6 +1420,9 @@ class SCTConfiguration(dict):
         dict(name="enable_argus", env="SCT_ENABLE_ARGUS", type=boolean,
              help="Control reporting to argus"),
 
+        dict(name="argus_test_id", env="SCT_ARGUS_TEST_ID", type=str,
+             help="Reuse existing argus test run id"),
```

Comment:
I think that here must be added more details because it is expected to be set by jenkins and not by user.